BUILDING.md

```diff
@@ -0,0 +1,98 @@
+# Compilation
+
+For more details and other workflows see the "Advanced guide" below.
+
+## Checking out the code
+
+```bash
+git clone https://github.com/libjxl/libjxl.git --recursive --shallow-submodules
+```
+
+This repository uses git submodules to handle some third party dependencies
+under `third_party`, that's why is important to pass `--recursive`. If you
```

Comment:
```suggestion
under `third_party`, that's why it is important to pass `--recursive`. If you
```

Reply:
this is now not fixed again in the pushed version...

Reply:
ugh my bad